lib/swagger-client.js

```diff
@@ -2674,8 +2694,7 @@ SwaggerOperation.prototype.urlify = function (args) {
     param = params[i];
     if(param.paramType === 'query') {
       var addition = '';
-      if (Array.isArray(param)) {
-        var j;   
+      if (Array.isArray(param)) { 
```

Comment:
Please use `_.isArray`. Array.isArray is not supported in all browsers we support